var/spack/repos/builtin/packages/py-svgwrite/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PySvgwrite(PythonPackage):
+    """A Python library to create SVG drawings."""
+
+    homepage = "https://pypi.org/project/svgwrite/"
+    url      = "https://pypi.io/packages/source/s/svgwrite/svgwrite-1.1.12.zip"
+
+    version('1.1.12', sha256='968c99f193f34f0fa7f0b3e82f49b93789c7c45cd89ce190480f16020d41ab79')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
Missing dependency:
```python
depends_on('py-pyparsing@2.0.1:', type=('build', 'run'))
```

Reply:
added -- will commit when we work out the details on the py-us dependency.